### PR TITLE
Refresh profile

### PR DESCRIPTION
**Pull request by leventdem**
- update localstorage if username or image is updated
- refresh profile info on update of profiledb

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -122,6 +122,20 @@ class App extends Component {
     if (prevProps.loading && !this.props.loading) {
       history.push(prevPath)
     }
+    if (!prevProps.currentUser && this.props.currentUser) {
+      const masq = getMasqInstance()
+      const handleUpdate = async () => {
+        await masq.updatePublicProfile()
```
Comment (G-Ray) [inline src/App.jsx:129]: good

File: src/lib/masq.js [hunk 2]
```diff
@@ -699,6 +715,25 @@ class Masq {
...
+   * @param {function} handleUpdate The function to apply for each update
+   * @returns The watcher
+   */
+  addWatcher (key, handleUpdate) {
+    const watcher = watch(this.profileDB, this.nonce, key, handleUpdate)
+    this.watchers.push(watcher)
+  }
+
+  destroyWatchers () {
+    this.watchers.forEach(watcher => {
```
Comment (G-Ray) [inline src/lib/masq.js:732]: need to check that watcher !== null || undefined

File: src/lib/masq.js [hunk 3]
```diff
@@ -699,6 +715,25 @@ class Masq {
...
+      if (watcher !== null || undefined) { watcher.destroy() }
```
Comment (G-Ray) [inline src/lib/masq.js]: (watcher !== null || watcher !=== undefined)

Comment (G-Ray) [reply src/lib/masq.js]: Or you can write if (!!watcher) { }

Comment (leventdem) [reply src/lib/masq.js]: ok done

**Review by G-Ray — APPROVED** [review]

**Pull request merged**